azurerm/internal/services/logic/validate/logic.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/logic/parse"
+	networkParse "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network/parse"
```

Comment:
Can we add these parse functions to the main network parse function to avoid having to alias the package here?

Reply:
Sorry, not sure if I understand here. I can wrap networkParse.SubnetID(subnetID) into the logic/parse package, but I think you were referring to some other solution. What do you mean by "main network parse function"? I think the collision is with "logic/parse" package. Or you mean to move parse.IntegrationServiceEnvironmentID to "network/parse"?